### PR TITLE
Regression protectli v121x

### PR DESCRIPTION
**Pull request by macpijan**
Testing execution on V1211:

```
DEVICE_IP=192.168.10.222 FW_FILE=protectli_v1211_v0.9.2.rom RTE_IP=192.168.10.227 CONFIG=protectli-v1211 SNIPEIT_NO=true ./scripts/regression.sh
```

### CODE REVIEW

**Comment by mkopec** [comment]
@macpijan rebased on latest develop

File: dasharo-performance/cpu-frequency.robot [hunk 1]
```diff
@@ -341,11 +341,14 @@ CPU Runs On Expected Frequency (Ubuntu)
         Log To Console    \n ----------------------------------------------------------------
         Log To Console    ${timer} min.
         @{frequencies}=    Get CPU Frequencies In Ubuntu
+        ${cpu_max_frequency_tol}=    Evaluate    ${CPU_MAX_FREQUENCY} * 1.125
+        ${cpu_min_frequency_tol}=    Evaluate    ${CPU_MIN_FREQUENCY} * 0.875
+        @{frequencies}=    Get CPU Frequencies In Ubuntu
```
Comment (macpijan) [inline dasharo-performance/cpu-frequency.robot:346]: Why getting freqs second time?

File: dasharo-performance/cpu-frequency.robot [hunk 2]
```diff
@@ -341,11 +341,14 @@ CPU Runs On Expected Frequency (Ubuntu)
         Log To Console    \n ----------------------------------------------------------------
         Log To Console    ${timer} min.
         @{frequencies}=    Get CPU Frequencies In Ubuntu
+        ${cpu_max_frequency_tol}=    Evaluate    ${CPU_MAX_FREQUENCY} * 1.125
+        ${cpu_min_frequency_tol}=    Evaluate    ${CPU_MIN_FREQUENCY} * 0.875
```
Comment (macpijan) [inline dasharo-performance/cpu-frequency.robot:345]: Why calculating this in loop over and over, it won't change.

File: TRUE [hunk 3]
```diff
@@ -0,0 +1 @@
+ffffffff
```
Comment (macpijan) [inline TRUE:1]: what's this?

File: dasharo-performance/cpu-frequency.robot (hunk 2 above)
Comment (wiktormowinski) [reply dasharo-performance/cpu-frequency.robot:345]: oh wait there are 2 loops, my bad

File: dasharo-performance/cpu-frequency.robot (hunk 1 above)
Comment (macpijan) [reply dasharo-performance/cpu-frequency.robot:346]: I am integrating and fixing both of these in https://github.com/Dasharo/open-source-firmware-validation/pull/361 branch

File: TRUE (hunk 3 above)
Comment (wiktormowinski) [reply TRUE:1]: truth, sir

**Comment by wiktormowinski** [comment]
all fixed now

File: os-config/ubuntu-credentials.robot [hunk 4]
```diff
@@ -3,7 +3,7 @@
 # across installations. At least hostname has been defined, and
 # username and hostname variables are used in the prompts. Before,
 # we needed to change them in prompts as well.
-${UBUNTU_USERNAME}=         ubuntu
+${UBUNTU_USERNAME}=         user
```
Comment (JanPrusinowski) [inline os-config/ubuntu-credentials.robot:6]: I think that the global standard is that the username is ubuntu. If on your platform it is different it should be changed on DUT to `ubuntu` and not in the code as it most propably will brake tests for other platforms.

File: platform-configs/protectli-v1210.robot [hunk 5]
```diff
@@ -4,7 +4,8 @@ Resource    include/protectli-v1x10.robot
 
 *** Variables ***
 ${DMIDECODE_PRODUCT_NAME}=      V1210
-
+${CPU_MAX_FREQUENCY}=           2900.0000
+${CPU_MIN_FREQUENCY}=           800.0000
```
Comment (JanPrusinowski) [inline platform-configs/protectli-v1210.robot:8]: v1211 should also recieve those values

File: dasharo-performance/cpu-frequency.robot (hunk 1 above)
Comment (wiktormowinski) [reply dasharo-performance/cpu-frequency.robot]: ok

File: platform-configs/protectli-v1210.robot (hunk 5 above)
Comment (wiktormowinski) [reply platform-configs/protectli-v1210.robot:8]: on it

File: os-config/ubuntu-credentials.robot (hunk 4 above)
Comment (wiktormowinski) [reply os-config/ubuntu-credentials.robot]: git add . incident
all good now

**Comment by macpijan** [comment]
INtegrated into: https://github.com/Dasharo/open-source-firmware-validation/pull/361

**Pull request closed without merging**